a3p-integration/proposals/b:restart-vats/package.json

```diff
@@ -0,0 +1,13 @@
+{
+    "agoricProposal": {
+        "type": "/agoric.swingset.CoreEvalProposal",
+        "source": "subdir"
```

Comment:
subdir? not submission?

I'm a little surprised to see the bundle checked in, but I guess that's the current state of the art?

Reply:
unfortunately yes